ballerina/tests/test.bal

```diff
@@ -0,0 +1,223 @@
+// AUTO-GENERATED FILE.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+import ballerina/test;
+import ballerina/oauth2;
+import ballerina/http;
```

Comment:
Re-order the imports according to this: https://learn-ballerina.github.io/best_practices/format_the_code.html?highlight=import#order-of-the-imports-in-a-bal-file